src/api/console_stringifier.py

```diff
@@ -34,8 +34,10 @@ def _modify_unit(self, unit: str) -> str:
         Returns the modified unit.
         """
         unit = (
-            unit.replace(r"\squared", "^2")
+            unit.replace(" ", "")
+            .replace(r"\squared", "^2")
             .replace(r"\cubed", "^3")
+            .replace("\\percent", "\\%")
```

Comment:
There's no need to escape `%` in the console, right?

Reply:
This is for another purpose: To still have a separator before %

Reply:
Do you have an example why we would need that?

Reply:
If a user writes `\kg\percent` (whyever he should do that), than it should result in `kg %`, not `kg%`.

Reply:
But that's already the case, isn't it? I get a space the console output

Reply:
See below

Reply:
thanks, I get it know ;) It's because of the line `unit_parts = re.split(r"[\\|\s]+", unit)` afterwards that we still need the escaping here